spec/scnnr/client_spec.rb

```diff
@@ -17,7 +17,6 @@
   let(:timeout) { 0 }
   let(:logger) { Logger.new('/dev/null') }
   let(:logger_level) { :info }
-
```

Comment:
Are the changes in this file needed? Is there any intention?

Reply:
Not needed :disappointed: That's something that has left after previous review changes